examples/react4xp.config.js

```diff
@@ -0,0 +1,81 @@
+//──────────────────────────────────────────────────────────────────────────────
+// A template version of this, with upated properties and explanations,
+//  can always be found in the react4xp NPM package:
+//   node_modules/react4xp/examples/react4xp.config.js after installing,
+//  or:
+//   https://github.com/enonic/enonic-react4xp/blob/master/examples/
+//    react4xp.config.js
+//──────────────────────────────────────────────────────────────────────────────
+
+module.exports = {
+
+  //────────────────────────────────────────────────────────────────────────────
+  // ENTRIES AND CHUNKING:
+  //────────────────────────────────────────────────────────────────────────────
+
+  // If nothing is added below, this is the default behaviour:
+  // - Default entry source folder is /site/, that is: src/main/resources/site/
+  //   and its subfolders.
+  // - Everything under react4xp root folder (src/main/resources/react4xp/) will
+  //   be considered chunks and will be bundled by webpack into a single
+  //   dependency imported by webpack: react4xp.<contenthash>.js
+  // - Everything under the react4xp root folder (src/main/resources/react4xp/)
+  //   will be considered non-entries: added files here can be imported by
+  //   react4xp entries, but otherwise unreachable from react4xp.
+  // - Default entryExtensions (file extensions to look for when finding entries
+  //   under OTHER entryDirs than /site/) are: jsx, js, tsx, ts, es6, es
+
+
+  // chunkDirs are folder names where importable, non-entry code is kept.
+  //  Comma-separated list of folder names, relative to
+  //  src/main/resources/react4xp/. Each folder added here will be bundled by
+  //  webpack into a separate dependency chunk with the same name as the folder,
+  //  and a hash: <foldername>.<contenthash>.js. This is good for grouping sets
+  //  of dependencies that belong together, or will frequently be requested from
+  //  the client together in some parts of a web page but not others, etc. The
+  //  react4xp root (src/main/resources/react4xp/) is the standard chunk
+  //  'react4xp', but you can add subfolders here to bundle them (and their
+  //  subfolders) in separate chunks. Or you can add relative paths to the
+  //  react4xp root to imported dependency code from elsewhere. Don't overlap
+  //  with entryDirs or /site/.
+  //
+  //chunkDirs: [''],
```

Comment:
Have you tested just uncommenting this line back in? Does the build handle an empty-string item as expected or does something unexpected happen? Would an empty array or an actual dirname example be better?